lib/msal-browser/test/interaction_client/NativeInteractionClient.spec.ts

```diff
@@ -764,6 +771,7 @@ describe("NativeInteractionClient Tests", () => {
                 name: ID_TOKEN_CLAIMS.name,
                 idTokenClaims: ID_TOKEN_CLAIMS,
                 nativeAccountId: mockWamResponse.account.id,
+                tenants: [ID_TOKEN_CLAIMS.tid],
```

Comment:
So much repetition around creating the test account.  Maybe a method to create these for each test?